core/domain/story_services.py

```diff
@@ -187,33 +189,21 @@ def apply_change_list(story_id, change_list):
         raise
 
 
-def _save_story(committer_id, story, commit_message, change_list):
-    """Validates a story and commits it to persistent storage. If
-    successful, increments the version number of the incoming story domain
-    object by 1.
+def validate_explorations(exp_ids, raise_error):
+    """Validates the explorations in the given story and checks whether they
+    are compatible on the mobile app.
```

Comment:
Either "compatible with" or "playable on". Probably the former.

Reply:
Done